docs/slotfilling.rst

```diff
@@ -3,12 +3,284 @@
 Slot Filling
 ============
 
-One of the most common conversation patterns
-is to collect a few pieces of information
-from a user in order to do something (book a restaurant, call an API, search a database, etc.).
-This is also called **slot filling**.
+One of the most common conversation patterns is to collect a few pieces of
+information from a user in order to do something (book a restaurant, call an
+API, search a database, etc.). This is also called **slot filling**. Our
+recommended approach for this is the FormAction, which is described in detail
```

Comment:
This line is repeated immediately below, so consider removing one.  But I think the second version is a bit more accurate because it mentions the succession of pieces of information (`in a row`)